API Explorer tweaks

spadgett opened the pull request:
* Move under Home nav section to make it more discoverable
* Add Namspaced and Description columns to table

cc @smarterclayton 
/assign @rhamilto 

@bmignano Let me know if you object to these changes

![Screen Shot 2019-08-13 at 3 03 52 PM](https://user-images.githubusercontent.com/1167259/62969591-9eba2e80-bddb-11e9-9870-419f6e56dd06.png)

rhamilto commented on frontend/public/style/mixin/_line-clamp.scss:
```diff
@@ -0,0 +1,6 @@
+@mixin co-line-clamp($lines) {
```
Add a default value so `$lines` is optional
```suggestion
@mixin co-line-clamp($lines: 3) {
```

spadgett commented:
@rhamilto updated

bmignano commented:
Makes sense to me! Curious if that text box filter needs to be `Filter by kind or group...` still since we have that Group filter dropdown now. Seems like we could just leave it at `Filter by kind...`?

spadgett commented:
@rhamilto fixed the linter error, another look?

smarterclayton commented:
I still am partial to "Explore" over "API Explorer".  If necessary we could
use a slightly longer H1 on the page "Explore All Resources"

On Tue, Aug 13, 2019 at 4:42 PM Sam Padgett <notifications@github.com>
wrote:

> @rhamilto <https://github.com/rhamilto> fixed the linter error, another
> look?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/console/pull/2344?email_source=notifications&email_token=AAI37J7TKJIOGDTYHEXILI3QEMMC5A5CNFSM4ILN54UKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD4G5P4Q#issuecomment-521000946>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAI37J2NTMKUUE3GT7HD6J3QEMMC5ANCNFSM4ILN54UA>
> .
>

spadgett commented:
Updated the nav item to "Explore" and changed the text filter to only filter kind.

<img width="1520" alt="Screen Shot 2019-08-13 at 6 58 21 PM" src="https://user-images.githubusercontent.com/1167259/62983180-7858bb00-bdfc-11e9-8078-d95274c5d5ea.png">

spadgett commented:
Added a fix to remember selected filters using the back button
